docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5798` Added a global DB upgrade that adds a new cache table and moves the cache data that should be unique per key there.
```

Comment:
We only add to the changelog changes that are useful to know for the user. In this case is a internal refactor and is not relevant for the user. Please remove it